Windows10DebloaterGUI.ps1

```diff
@@ -270,7 +270,7 @@ $Debloat.ForeColor               = [System.Drawing.ColorTranslator]::FromHtml("#
 
 $CustomizeBlacklist             = New-Object system.Windows.Forms.Button
 $CustomizeBlacklist.FlatStyle   = 'Flat'
-$CustomizeBlacklist.text        = "CUSTOMISE BLOCKLIST"
+$CustomizeBlacklist.text        = "CUSTOMIZE BLOCKLIST"
```

Comment:
Fixed the typo for consistency, in the form it is labelled: Customize Allowlist and Blocklist